client/src/components/Toolshed/RepositoryDetails/InstallationButton.vue

```diff
@@ -19,7 +19,16 @@ import BootstrapVue from "bootstrap-vue";
 Vue.use(BootstrapVue);
 
 export default {
-    props: ["installed", "status"],
+    props: {
+        installed: {
```

Comment:
As you mentioned, this property is not used anymore. It should be removed from `Index.vue` too. Thanks a lot.